install_custom_module_name/after/Pods/Pods.xcodeproj.yaml

```diff
@@ -109,16 +109,15 @@ Targets:
           DYLIB_INSTALL_NAME_BASE: '@rpath'
           MODULEMAP_FILE: Target Support Files/Pods/Pods.modulemap
           CURRENT_PROJECT_VERSION: '1'
+          OTHER_LIBTOOLFLAGS: ''
```

Comment:
It does override the flags form the XCConfings, [intentionally](https://github.com/CocoaPods/CocoaPods/blob/master/lib/cocoapods/installer/target_installer/aggregate_target_installer.rb#L35-47).